content/articles/2018-05-dplyr-0-7-5.Rmd

```diff
@@ -0,0 +1,161 @@
+---
+title: dplyr 0.7.5
+slug: dplyr-0.7.5
+description: >
+    dplyr 0.7.5 has reached CRAN. This is mostly a bugfix release with two important new changes.
+date: 2018-05-25
+author: Kirill Müller, Romain François, Hadley Wickham
+photo:
+  url: https://unsplash.com/photos/aqrIcYonB-o
+  author: Phúc Long
+categories: [package]
+---
+
+
+```{r setup, include = FALSE}
+options(crayon.enabled = TRUE)
+options(pillar.bold = TRUE)
+
+library(dplyr)
+knitr::opts_chunk$set(collapse = TRUE, comment = pillar::style_subtle("#>"))
+
+colourise_chunk <- function(type) {
+  function(x, options) {
+    sprintf('<pre class="knitr %s">%s</pre>\n',
+      tolower(options$engine),
+      fansi::sgr_to_html(htmltools::htmlEscape(x))
+    )
+  }
+}
+
+knitr::knit_hooks$set(
+  output = colourise_chunk("output"),
+  message = colourise_chunk("message"),
+  warning = colourise_chunk("warning"),
+  error = colourise_chunk("error")
+)
+```
+
+We’re excited to announce version 0.7.5 of the *dplyr* package, the grammar of data manipulation in the tidyverse.  This minor release features the move to *tidyselect*, minor features like scoped operations on grouped data frames and support for raw vectors, and a number of bug fixes.  Please see the [release notes](https://github.com/tidyverse/dplyr/releases/tag/v0.7.5) for the full list of improvements and bug fixes.
+
+The next planned release of *dplyr* will be a feature release: Romain, author of the data frame backend for *dplyr* and coauthor of the *Rcpp* package, has joined the team and hit the ground running.  He has implemented many of the features described in this blog post, many more are already available in the development version.  Welcome (back), Romain, looking forward to a great time!
+
+
+## tidyselect
+
+*dplyr* always supported selecting by name, excluding columns, selecting by range, by match, or by position:
+
+```{r select}
+tbl <- data.frame(a1 = 1, a2 = 2, a3 = 3, b = "x")
+
+tbl %>%
+  select(a1, a2)
+tbl %>%
+  select(-b)
+tbl %>%
+  select(a1:a3)
+tbl %>%
+  select(starts_with("a"))
+tbl %>%
+  select(2:4)
+vars <- syms(c("a2", "b"))
+tbl %>%
+  select(!!!vars)
+```
+
+Last year, the core code that provides this functionality out of *dplyr* into the fairly new *tidyselect* package.  Selecting columns in a data frame (or items in a character vector, for that matter) is a common task in many other situations.  The *tidyselect* package offers a consistent and convenient interface with full support for quasiquotation, and is used by more than 20 packages, and now also by *dplyr*.  Internally, the `select()` calls above are translated into the following *tidyselect* operations:
```

Comment:
I think the word "moved" (or "was moved", "we moved") is missing from that first sentence.